<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionScene</class>
 <widget class="QWidget" name="QuestionScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="question">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="player1Label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>14</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>player1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rebusAnswer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>151</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="answer1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>a)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="answer2">
      <property name="text">
       <string>b)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="answer3">
      <property name="text">
       <string>c)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="answer4">
      <property name="text">
       <string>d)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
